src/core/graph.js

```diff
@@ -209,6 +209,15 @@ type CachedOrder = {|
   +modificationCount: number,
 |};
 
+/**
+ * Specifies how to contract the graph, collapsing several old nodes
```

Comment:
s/the graph/a graph/: This is a type definition; there is no “`this`”
context.

Reply:
Done.